test/test.go

```diff
@@ -28,6 +28,80 @@ func testEmbed(g abstract.Group, rand cipher.Stream, points *[]abstract.Point,
 	*points = append(*points, p)
 }
 
+func testPointSet(g abstract.Group, rand cipher.Stream) {
+	N := 1000
+	null := g.Point().Null()
+	for i := 0; i < N; i++ {
```

Comment:
Hmm, I agree that reproducibility is indeed a *very* good argument against random tests. On the other hand we increase the coverage (which is a treadeoff between testing against only one totally deterministically created point and testing against all). 
@ineiti if you are still convinced that we should keep it like this I would still prefer to leave it as is.